.jscsrc

```diff
@@ -6,6 +6,7 @@
                   "./examples/simple/node_modules/**",
                   "./examples/complex/node_modules/**",
                   "./examples/webpack/node_modules/**",
+		  "./examples/edge/**",
```

Comment:
I added a new example "edge" to test the redirect functionality. Figured since everything changed, and the example uses `require(babel/register)` I may as well move to ES6. Conflicts with JSCS, see https://github.com/jscs-dev/node-jscs/issues/1573, specifically `esprima-fb` issue.